app/router.js

```diff
@@ -61,6 +61,7 @@ Router.map(function () {
     this.route('build', { path: '/builds/:build_id', resetNamespace: true }, function () {
       this.route('config');
     });
+    this.route('config');
```

Comment:
I'd call this route a bit more semantically, something like `trigger-build`. Currently it reads as "Repository Config", which, as far as I understand, it isn't.

Reply:
i see your concern.

"config" certainly is an overloaded term, but it's more than "just" triggering builds. it's also about trying out and previewing request configs (potentially adding more functionality in the future).

how about we make it a nested route `repo/:slug/request/config`? i'm not sure if `repo/:slug/build/config` is taken by the build id.

Reply:
Do you know what kind of functionality is going to be added there in the future? Probably we can find a name based on that? The nested route sounds good to me, but I'd rather suggest something like `repo/:provider/:owner/:name/requests/new/config` - as we have already `requests` route in plural form and adding one in singular form looks a bit inconsistent. What do you think?

Reply:
i think essentially this could be a "home" place for everything around configuring your build. trying out how matrix expansion works (using the preview feature), trying out how configs merge into each other (for either api requests or config imports), actually submitting those as build requests (i.e. "trigger build").

one bigger idea is to add a "build config wizard" which asks you for your language, and then from there takes you through a bunch of questions, helping newcomers build up their `.travis.yml` file.

obviously this is all pretty unknown today, but i think it would be good to keep the route as generic as possible. even the `new` in `requests/new/config` seems rather specific ...

having said that i am still undecided and going back and forth on what i think the best route naming would be ... 🤔

Reply:
i have added that question to a new section "to be discussed/resolved" on https://github.com/travis-ci/travis-web/pull/2399 in order to keep track of this